docs/5.x/requirements.md

```diff
@@ -88,27 +88,43 @@ These ports are required during initial installation and can be closed after the
 | 61008-61010, 61022-61024 | HTTPS     | Installer agent ports     |
 | 4242                     | TCP       | Bandwidth checker utility |
 
+### Default Subnets
 
-#### Cluster Ports
+By default Gravity clusters are configured to use the following network subnets:
+
+| Subnet          | Description               |
+|--------------- -|---------------------------|
+| 10.244.0.0/16   | Pod IPv4 addresses        |
+| 10.100.0.0/16   | Services IPv4 addresses   |
+
+### Cluster Ports
 
 These ports are used for Cluster operation and should be open between cluster nodes:
 
-| Port                    | Protocol                                | Description                               |
-|-------------------------|-----------------------------------------|-------------------------------------------|
-| 53                      | TCP and UDP                             | Internal cluster DNS                      |
-| 8472                    | VXLAN (UDP encapsulation)               | Overlay network                           |
-| 7496, 7373              | TCP                                     | Serf (Health check agents) peer to peer   |
-| 7575                    | TCP                                     | Cluster status gRPC API                   |
-| 2379, 2380, 4001, 7001  | HTTPS                                   | Etcd server communications                |
-| 6443                    | HTTPS                                   | Kubernetes API Server                     |
-| 30000 - 32767           | HTTPS (depend on the services deployed) | Kubernetes internal services range        |
-| 10248 - 10250, 10255    | HTTPS                                   | Kubernetes components                     |
-| 5000                    | HTTPS                                   | Docker registry                           |
-| 3022-3025               | SSH                                     | Teleport internal SSH control panel       |
-| 3080                    | HTTPS                                   | Teleport Web  UI                          |
-| 3008-3012               | HTTPS                                   | Internal Gravity services                 |
-| 32009                   | HTTPS                                   | Gravity Cluster/OpsCenter Admin panel UI  |
-| 3012                    | HTTPS                                   | Gravity RPC  agent                        |
+| Port                    | Protocol - Layer 4 | Protocol - Layer 5              | Source      | Destination | Description                              |
```

Comment:
Not sure I follow what exactly is meant by "source" and "destination" here. Can you elaborate?